index.bs

```diff
@@ -762,12 +807,12 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
     1. Let |credentialList| be a new [=list=].
 
-    1. If <code>|publicKeyOptions|.{{PublicKeyCredentialRequestOptions/allowList}}</code> [=list/is not empty=], execute a
+    1. If <code>|options|.{{PublicKeyCredentialRequestOptions/allowList}}</code> [=list/is not empty=], execute a
```

Comment:
This is as far as I got today. If the rest is just editorial cleanups, the stuff I looked at is good to merge.